homeassistant/components/matter/light.py

```diff
@@ -43,6 +43,18 @@
 }
 DEFAULT_TRANSITION = 0.2
 
+# there's a bug in (at least) Espressif's implementation of light transitions
+# on devices based on Matter 1.0. Mark potential devices with this issue.
+# https://github.com/home-assistant/core/issues/113775
+# vendorid (attributeKey 0/40/2)
+# productid (attributeKey 0/40/4)
+# hw version (attributeKey 0/40/8)
+# sw version (attributeKey 0/40/10)
+TRANSITION_BLOCKLIST = (
```

Comment:
Make this a set so that the membership test is fast.

Reply:
oops, too late its already merged. I expect a few more reports of id's in that issue report so once we add more defs, we can make it a set.